### PR TITLE
Upgrade Dub version + use workspace API keys

### PR DESCRIPTION
**Pull request by steven-tey**
This PR upgrades `dub` to the latest version and uses the new [Workspace API Keys](https://dub.co/blog/workspace-api-keys) feature that we launched recently.

CleanShot 2024-07-02 at 08 53 07

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,5 +82,6 @@
     "stylelint-webpack-plugin": "^3.3.0",
     "tailwindcss": "^3.4.3",
     "typescript": "^5.4.2"
-  }
+  },
+  "packageManager": "pnpm@9.3.0+sha512.ee7b93e0c2bd11409c6424f92b866f31d3ea1bef5fbe47d3c7500cdc3c9668833d2e55681ad66df5b640c61fa9dc25d546efa54d76d7f8bf54b13614ac293631"
```
Comment (joaopcm) [inline package.json:86]: Should this be added?

Comment (steven-tey) [reply package.json:86]: @joaopcm good catch – can probably leave that out, it was autogenerated when I ran `pnpm i` 😅

Comment (joaopcm) [reply package.json:86]: yeah, I hate when my CLI/IDE adds this automatically as well, lol

great! thanks!

**Comment by samuelkraft** [comment]
Thanks @steven-tey!
I checked out your branch locally as it fails on vercel due to the pnpm addition instead of using npm - but it fails when I try to shorten a link. Any idea why?

Sam 2024-07-02 at 8  23 52@2x

**Comment by steven-tey** [comment]
@samuelkraft I just simplified the PR by switching to NPM and it looks like the preview URL is working now: https://ray-gi2t7dair-raycastapp.vercel.app

If it still doesn't work in your local env, can you double-check to see if you're using the correct API key (workspace API keys start with `dub_`)?

**Review by samuelkraft — APPROVED** [review]
great 🙌

**Pull request merged**